@xen-orchestra/web/src/components/hosts/panel/card/HostHardwareSpecificationsCard.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <UiCard>
+    <UiCardTitle>
+      {{ t('hardware-specifications') }}
+    </UiCardTitle>
+    <div class="content">
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('manufacturer-info') }}</template>
+        <template #value>{{ manufacturerInfo }}</template>
+        <template #addons>
+          <VtsCopyButton :value="manufacturerInfo" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('core-socket') }}</template>
+        <template #value>{{ coreSocketInfo }}</template>
+        <template #addons>
+          <VtsCopyButton :value="coreSocketInfo" />
+        </template>
+      </VtsCardRowKeyValue>
+    </div>
+  </UiCard>
+</template>
+
+<script lang="ts" setup>
+import type { XoHost } from '@/types/xo/host.type'
+import VtsCardRowKeyValue from '@core/components/card/VtsCardRowKeyValue.vue'
+import VtsCopyButton from '@core/components/copy-button/VtsCopyButton.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiCardTitle from '@core/components/ui/card-title/UiCardTitle.vue'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { host } = defineProps<{
+  host: XoHost
+}>()
+
+const { t } = useI18n()
+
+const manufacturerInfo = computed(
+  () =>
+    (host.bios_strings['system-manufacturer'] ?? '') +
+    (host.bios_strings['system-product-name'] ? ` (${host.bios_strings['system-product-name']})` : '')
+)
+
+const coreSocketInfo = computed(() => {
+  const cores = host.cpus.cores
+  const sockets = host.cpus.sockets
```

Comment:
```suggestion
  const cores = host.cpus.cores ?? 0
  const sockets = host.cpus.sockets ?? 0
```

and in XoHost type you can change this : 
```
 cpus: {
    cores?: number
    sockets?: number
  }
  ```
  
  and in `HostDashboardCpuProvisionning` you can change this line too 
  ```
  const cpusCount = computed(() => host.cpus.cores ?? 0)
```